tests/test_config.py

```diff
@@ -50,16 +52,11 @@ def test_config_env_access_error(config):
         _ = config.DOES_NOT_EXIST
 
 
-def test_config_env_access_raise_error_if_network_config_not_json(monkeypatch, config):
-    monkeypatch.setenv("ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG", "test")
-    with pytest.raises(InvalidNetworkConfigurationError, match="Expecting value"):
-        _ = config.ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG
-
-
-def test_config_env_access_raise_error_if_network_config_is_missing(monkeypatch, config):
-    monkeypatch.delenv("ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG")
-    with pytest.raises(
-        InvalidNetworkConfigurationError,
-        match="the JSON object must be str, bytes or bytearray, not NoneType",
-    ):
-        _ = config.ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG
+def test_config_env_access_network_config_success(monkeypatch, config):
+    assert config.ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG == {
+        "awsvpcConfiguration": {
+            "subnets": ["subnet-abc123"],
```

Comment:
@jonavellecuerdo it is more likely than not that the "subnets" value will be a list of subnets, not a single subnet. Would it make sense for your unit test to list two subnets instead of one here?

Reply:
Thank you for the suggestion! See this commit https://github.com/MITLibraries/alma-sapinvoices-ui/pull/83/commits/c858fd122d3337e7d3016e81940011f6e50229e0 for more details. 

**Note:** [`boto3.ECSClient.run_task`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs/client/run_task.html) requires these network configuration dictionary to pass these values as a list. The call to `split` the string by the comma (`,`) will return a list, even for single values, so I think [this logic](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/83/files#diff-966b49fcc05ab61f6b2ef9d13713ceb9da0b5d825b9ef5cdb16961b931e6bfd6R41-R42) is fine to leave as-is.